src/command.rs

```diff
@@ -199,7 +200,7 @@ fn cmd_thread(kill_rx: oneshot::Receiver<()>, died_tx: oneshot::Sender<()>,
 
   core.run(cmd_loop).ok();
 
-  died_tx.complete(());
+  died_tx.send(()).expect("Input thread died prematurely.");
```

Comment:
If this send fails it would be the main thread that has gone wonky, so change "Input" to "Main" please.